demo/config.js

```diff
@@ -586,6 +567,32 @@ shakaDemo.Config = class {
         .addBoolInput_('Don\'t choose codecs',
             'streaming.dontChooseCodecs');
     this.addRetrySection_('streaming', 'Streaming Retry Parameters');
+    this.addLiveSyncSection_();
+  }
+
+  /** @private */
+  addLiveSyncSection_() {
+    const docLink = this.resolveExternLink_('.LiveSync');
```

Comment:
This links to the expected URL, but I couldn't figure out how to get docs to build properly to include a section for live sync in there.

Reply:
An @exportDoc was missing